### PR TITLE
Extend ApiRequestError with HttpError details

### PR DESCRIPTION
**Pull request by MaxRis**

### CODE REVIEW

**Comment by shcheklein** [comment]
@MaxRis CI failed?

File: pydrive2/files.py [hunk 1]
```diff
@@ -27,6 +28,13 @@ class FileNotUploadedError(RuntimeError):
 class ApiRequestError(IOError):
     """Error while making any API requests."""
 
+    def __init__(self, http_error):
```
Comment (shcheklein) [inline pydrive2/files.py]: I think we should still initialize it in a regular way - it should support a regular `ctr` with cause and call `super`

Comment (MaxRis) [reply pydrive2/files.py]: @shcheklein have updated to `chain` it `from HttpError` (so __cause__ is set)

**Comment by shcheklein** [comment]
CI still fails

File: pydrive2/files.py (hunk 1 above)
Comment (shcheklein) [reply pydrive2/files.py]: I don't see it, super is called w/o cause

**Comment by MaxRis** [comment]
@shcheklein , hopefully, PR should be ready now, take a look, please, thanks!

**Comment by shcheklein** [comment]
Looks good to me. Can we add a test for this? Or find one that already tests exception and check for the error propagation at least.

**Comment by MaxRis** [comment]
@shcheklein have added a separate test for it

**Comment by shcheklein** [comment]
@efiop @MaxRis could you guys release a new version please?

**Comment by efiop** [comment]
@shcheklein 1.4.8 is out.

**Comment by efiop** [comment]
@MaxRis Oh, thanks for the heads up! Looking into it...

**Comment by efiop** [comment]
@MaxRis It was a hickup on travis' side. 1.4.8 is out.

**Comment by MaxRis** [comment]
@efiop thank you!

**Pull request merged**